src/anndata/_core/views.py

```diff
@@ -274,6 +274,19 @@ def drop(self, *args, inplace: bool = False, **kw):
         with view_update(*self._view_args) as df:
             df.drop(*args, inplace=True, **kw)
 
+    def __setattr__(self, key: str, value: Any):
+        if key == "index":
+            warnings.warn(
+                f"Trying to modify {key} from attribute `.{self._view_args.attrname}` of view, "
```

Comment:
```suggestion
                f"Trying to modify {key} of attribute `.{self._view_args.attrname}` of view, "
```

I think this is the right grammar? But you’re the native speaker, so idk!

Reply:
I think both are ok, but yours is probably more fluid yea